dts/bindings/sensor/memsic,mc3419.yaml

```diff
@@ -21,3 +31,16 @@ properties:
       has two interrupt pins.By default the interrupt are routed
       to interrupt pin 1, by enabled this property interrupt are
       routed to interrupt pin 2.
+
+  lpf-fc-sel:
+    type: int
+    default: 0
+    description: |
+      Enable and select LPF cutoff frequency for a given IDR (Input Data Rate).
+      Default is LPF disabled (reset value of the register).
+    enum:
+      - 0  # MC3419_LPF_DISABLE
+      - 9  # MC3419_LPF_EN_WITH_IDR_BY_4p255_FC
+      - 10 # MC3419_LPF_EN_WITH_IDR_BY_6_FC
+      - 11 # MC3419_LPF_EN_WITH_IDR_BY_12_FC
+      - 13 # MC3419_LPF_EN_WITH_IDR_BY_16_FC
```

Comment:
Move the comments describing the enumerations into the description. This will allow the content to be included in the generated documentation